### DISCUSSION TITLE
How to open on ubuntu?

### DISCUSSION THREAD

**Discussion by wyteroze** [Leather128/LeathersFunkinEngine#247] [Q&A] [answered]
How do i open leather engine on ubuntu? When i finished compiling it, leather engine automatically opened and everything went fine, but when i went to the export folder to open it again, it just shows as a little gear icon and says it's a "shared library" file (the software install doesnt work and just says "failed to install file: not supported"
![image](https://user-images.githubusercontent.com/82071274/170536631-faf1f1ce-7484-4ba9-902a-65406733b8d7.png)
![image](https://user-images.githubusercontent.com/82071274/170536757-438739d5-5ad1-46b3-92e4-56eff04b5645.png)

**Comment by Beef64** [comment, marked as answer]
Not sure how you didn't know this already, but all you have to do is make the file able to be run as an executable, usually with `chmod +x [path to file goes here]` and replace [path to file goes here] with the file path to the Leather Engine executable.
